homeassistant/components/advantage_air/climate.py

```diff
@@ -116,6 +148,25 @@ def fan_mode(self) -> str | None:
         """Return the current fan modes."""
         return ADVANTAGE_AIR_FAN_MODES.get(self._ac["fan"])
 
+    @property
+    def preset_mode(self) -> str:
+        """Return the current preset mode."""
+        if self._ac.get(ADVANTAGE_AIR_MYAUTO_ENABLED):
+            return ADVANTAGE_AIR_MYAUTO
+        if self._ac.get(ADVANTAGE_AIR_MYTEMP_ENABLED):
+            return ADVANTAGE_AIR_MYTEMP
+        return ADVANTAGE_AIR_MYZONE
+
```

Comment:
```suggestion
```

Lets drop this since `ClimateEntityFeature.PRESET_MODE` isn't set as a supported feature